app/repository/ReliefsMongoRepository.scala

```diff
@@ -152,7 +160,7 @@ class ReliefsReactiveMongoRepository(mongo: () => DB, val metrics: ServiceMetric
     val timerContext = metrics.startTimer(MetricsEnum.RepositoryDeleteRelief)
```

Comment:
Just `fetchReliefs` at line 150 `collection.find(...)` that is not wrapped in `Mdc.preservingMdc`

Reply:
I didn't add it here because there is no logging off the back of the return from mongo.